storages/backends/gcloud.py

```diff
@@ -249,10 +252,17 @@ def get_created_time(self, name):
         return created if setting('USE_TZ') else timezone.make_naive(created)
 
     def url(self, name):
-        # Preserve the trailing slash after normalizing the path.
+        """
+        Return public url or a signed url for the Blob.
+        This DOES NOT check for existance of Blob - that makes codes too slow
+        for many use cases.
+        """
         name = self._normalize_name(clean_name(name))
-        blob = self._get_blob(self._encode_name(name))
-        return blob.public_url
+        blob = self.bucket.blob(self._encode_name(name))
+        if self.default_acl == 'publicRead':
```

Comment:
My nitpick on style would be to either:

a) put a blank line above the `if` and eliminate the one below it
b) use a ternary with no newline (a bit ugly i suppose with the function call)

Reply:
Fixed.